sqlstore/src/main/resources/db/migration/V8092__qcPassedDetails_to_detailedQcStatus.sql

```diff
@@ -0,0 +1,37 @@
+-- add basic defaults
+
+INSERT INTO QcPassedDetail (status, description, noteRequired, createdBy, creationDate, updatedBy, lastUpdated) 
+  SELECT TRUE, 'Ready', FALSE, (SELECT userId FROM User WHERE loginName = 'admin'), NOW(), (SELECT userId FROM User WHERE loginName = 'admin'), NOW() 
+  WHERE NOT EXISTS (SELECT description FROM QcPassedDetail WHERE description = 'Ready' LIMIT 1);
+INSERT INTO QcPassedDetail (status, description, noteRequired, createdBy, creationDate, updatedBy, lastUpdated) 
+  SELECT FALSE, 'Failed: QC', FALSE, (SELECT userId FROM User WHERE loginName = 'admin'), NOW(), (SELECT userId FROM User WHERE loginName = 'admin'), NOW() 
+  WHERE NOT EXISTS (SELECT description FROM QcPassedDetail WHERE description = 'Failed: QC' LIMIT 1);
+
+-- remove unnecessary option, as it will be the null case.
+ALTER TABLE DetailedSample DROP FOREIGN KEY `FKa2t38wms0eer896xo4fw76tw0`;
+DELETE FROM QcPassedDetail WHERE description = 'Not Ready';
```

Comment:
This migration will fail when re-adding the foreign key if anything uses that value. Skip dropping the foreign key and first do:
```
UPDATE DetailedSample SET qcPassedDetailId = NULL WHERE qcPassedDetailId = (
  SELECT qcPassedDetailId FROM QcPassedDetail WHERE description = 'Not Ready';
);
```